Scene/RasterCamera.h

```diff
@@ -64,6 +68,22 @@ class RasterCamera
 			m_cameraPosition -= m_cameraUp * velocity;
 		if (direction == DOWN)
 			m_cameraPosition += m_cameraUp * velocity;
+		if (direction == YAWLEFT) {
+			m_cameraYaw += 1.0f;
```

Comment:
I would move this value 1.0f into a variable called `float speed`. so later if we want to change all at once we can just edit `speed`